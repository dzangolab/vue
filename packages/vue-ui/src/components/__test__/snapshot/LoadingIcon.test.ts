import { mount } from "@vue/test-utils";
import { describe, expect, it } from "vitest";

import LoadingIcon from "../../LoadingIcon.vue";

describe("LoadingIcon", () => {
  it("matches snapshot", () => {
    const wrapper = mount(LoadingIcon, {
      props: {
        label: "Login",
      },
    });

    expect(wrapper.element).toMatchSnapshot();
  });
});
