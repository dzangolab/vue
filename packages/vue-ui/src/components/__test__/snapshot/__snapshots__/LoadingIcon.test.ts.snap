// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`LoadingIcon > matches snapshot 1`] = `
<div
  class="loading"
  data-v-20744668=""
  label="Login"
>
  Loading
</div>
`;
